.github/actions/dockerfiles/Dockerfile.signer.debian

```diff
@@ -0,0 +1,24 @@
+FROM rust:1.81.0-slim-bookworm as build
+
+# Install dependencies.
+RUN apt-get update
+RUN apt-get install -y \
+    libclang-dev \
+    git \
+    pkg-config \
+    libssl-dev \
+    make \
+    protobuf-compiler \
+    npm \
+    default-jre
+RUN npm install -g pnpm@9
+RUN npm install -g @openapitools/openapi-generator-cli
+
+WORKDIR /code/sbtc
+COPY . .
+RUN make install && make build
+
+# Create Docker image to run the signer.
+FROM debian:bookworm-slim AS signer
+COPY --from=build /code/sbtc/target/debug/signer /usr/local/bin/signer
+ENTRYPOINT ["/usr/local/bin/signer --config /signer-config.toml --migrate-db"]
```

Comment:
For this image, i would leave this as CMD for troubleshooting purposes. 
for a production image, entrypoint is fine though (although i personally still prefer CMD since it allows some customization)

Reply:
I'm happy to move this to CMD, but for context we are only expecting these to be used for production

Reply:
Looking at the differences between these I agree, `CMD` makes sense.

Reply:
Done.